data/korath/korath ships.txt

```diff
@@ -851,6 +851,213 @@ ship "Korath World-Ship" "Korath World-Ship C"
 	turret 91 290 "Korath Warder"
 
 
+ship "Korath Haven"
+	sprite "ship/korath haven"
+	"uncapturable"
+	attributes
+		category "Heavy Freighter"
+		cost 27690000
+		shields 196420
+		hull 320690
+		"required crew" 1940
+		"bunks" 10920
+		"mass" 12000
+		"drag" 84
+		"heat dissipation" .22
+		"fuel capacity" 1700
+		"cargo space" 1864
+		"outfit space" 2630
+		"weapon capacity" 1181
+		"engine capacity" 516
```

Comment:
Probably need to raise outfit and engines for edits below but I'm to tired for maths